doc/submitter_guide.md

```diff
@@ -60,19 +60,18 @@ dashboard](https://kcidb.kernelci.org/).
 2\. Install KCIDB
 -----------------
 
-Pick a KCIDB [release][releases]. Latest would normally be best.
```

Comment:
Please remove the `releases` link reference at the bottom of the file as well.

Reply:
done